manim/mobject/tabular.py

```diff
@@ -0,0 +1,1114 @@
+        if self.include_outer_lines:
+            anchor = self.get_columns()[0].get_left()[0] - 0.5 * self.h_buff
+            line = Line(
+                [anchor, anchor_top, 0], [anchor, anchor_bottom, 0], **self.line_config
+            )
+            line_group.add(line)
+            self.add(line)
+            anchor = self.get_columns()[-1].get_right()[0] + 0.5 * self.h_buff
+            line = Line(
+                [anchor, anchor_top, 0], [anchor, anchor_bottom, 0], **self.line_config
+            )
+            line_group.add(line)
+            self.add(line)
+        for k in range(len(self.mob_table[0]) - 1):
+            anchor = self.get_columns()[k + 1].get_left()[0] + 0.5 * (
+                self.get_columns()[k].get_right()[0]
+                - self.get_columns()[k + 1].get_left()[0]
+            )
+            line = Line(
+                [anchor, anchor_bottom, 0], [anchor, anchor_top, 0], **self.line_config
+            )
+            line_group.add(line)
+            self.add(line)
+        self.vertical_lines = line_group
+        return self
+
+    def get_horizontal_lines(self) -> VGroup:
+        """Return the horizontal lines of the table.
+
+        Returns
+        --------
+        :class:`~.VGroup`
+            :class`~.VGroup` containing all the horizontal lines of the table.
+
+        Examples
+        --------
+
+        .. manim:: GetHorizontalLinesExample
+            :save_last_frame:
+
+            class GetHorizontalLinesExample(Scene):
+                def construct(self):
+                    table = Tabular(
+                        [["First", "Second"],
+                        ["Third","Fourth"]],
+                        row_labels=[Text("R1"), Text("R2")],
+                        col_labels=[Text("C1"), Text("C2")])
+                    table.get_horizontal_lines().set_color(RED)
+                    self.add(table)
+        """
+        return self.horizontal_lines
+
+    def get_vertical_lines(self) -> VGroup:
+        """Return the vertical lines of the table.
+
+        Returns
+        --------
+        :class:`~.VGroup`
+            :class:`~.VGroup` containing all the vertical lines of the table.
+
+        Examples
+        --------
+
+        .. manim:: GetVerticalLinesExample
+            :save_last_frame:
+
+            class GetVerticalLinesExample(Scene):
+                def construct(self):
+                    table = Tabular(
+                        [["First", "Second"],
+                        ["Third","Fourth"]],
+                        row_labels=[Text("R1"), Text("R2")],
+                        col_labels=[Text("C1"), Text("C2")])
+                    table.get_vertical_lines()[0].set_color(RED)
+                    self.add(table)
+        """
+        return self.vertical_lines
+
+    def get_columns(self) -> List[VGroup]:
+        """Return columns of the table as a list of :class:`~.VGroup`.
+
+        Returns
+        --------
+        List[:class:`~.VGroup`]
```

Comment:
**(comment/suggestion)**: it actually returns a `VGroup` of vgroups. not a list of vgroups,  VGroup[VGroup] might be valid here.